### PR TITLE
Made forms for changing some user information (Name and Password)

### PR DESCRIPTION
**Pull request by Weeezzt**
---

### Pull Request Summary

**Feature Enhancements:**

1. **Profile Page Updates:**
   - Implemented forms for updating user information:
     - Separate form for updating the user's name.
     - Separate form for updating the user's password.

2. **UI Improvements:**
   - Refined the footer and header for better responsiveness and aesthetics on both mobile and desktop.
   - Changed the background color to `stone-800` from `stone-700` for improved visual consistency.

3. **Bug Fixes:**
   - Resolved issues where certain pages were causing the footer to misalign.

---

### CODE REVIEW

File: kinogoteborg/src/app/components/profile/Settings.jsx [hunk 1]
```diff
@@ -1,7 +1,137 @@
...
+
   return (
-    <div className="h-full col-span-6 w-full bg-stone-700 flex flex-col items-center gap-8">
+    <div className="h-full col-span-6 w-full bg-stone-800 flex flex-col items-center gap-8 mb-10">
       <h1 className="text-2xl text-stone-200 text-center mt-8">Settings</h1>
+      <div className="flex w-full h-full items-center md:items-baseline justify-center md:h-3/5 flex-col md:flex-row gap-5 lg:gap-20">
+        <div className="border-2 border-solid border-stone-500 bg-zinc-800 w-3/4 md:w-3/5 xl:w-1/3 min-h-32 shadow-md rounded-md text-stone-200 p-8 m-5">
+          <h2 className="mb-5 text-xl font-bold">Change name</h2>
+          <form className="flex flex-col gap-4" onSubmit={onSubmitNameChange}>
+            <label htmlFor="name">New name</label>
```
Comment (MatsJoh50) [inline kinogoteborg/src/app/components/profile/Settings.jsx:87]: change to first + lastname or clerify structure of the name.

File: kinogoteborg/src/app/components/profile/Settings.jsx [hunk 2]
```diff
@@ -1,7 +1,137 @@
...
+            {error1 && (
+              <div className="bg-red-500 text-white w-fit text-sm py-1 px-3 rounded-md mt-2">
+                {error1}
+              </div>
+            )}
+          </form>
+        </div>
+        <div className="border-2 border-solid border-stone-500 bg-zinc-800 w-3/4 md:w-3/5 xl:w-1/3 min-h-32 shadow-md rounded-md text-stone-200 p-8 m-5">
+          <h2 className="mb-5 text-xl font-bold">Change Password</h2>
+          <form className="flex flex-col gap-4" onSubmit={onSubmitNewPassword}>
```
Comment (MatsJoh50) [inline kinogoteborg/src/app/components/profile/Settings.jsx:107]: Add old password for increased security for the user?

**Review by MatsJoh50 — APPROVED** [review]
left comments, take propper steps forward now.

File: kinogoteborg/src/app/components/profile/Settings.jsx (hunk 2 above)
Comment (Kookiie88) [reply kinogoteborg/src/app/components/profile/Settings.jsx:107]: That is a good suggestion, we will add it in a future update.

File: kinogoteborg/src/app/components/profile/Settings.jsx (hunk 1 above)
Comment (Kookiie88) [reply kinogoteborg/src/app/components/profile/Settings.jsx:87]: For now the name is not used in any way so i dont think its needed to split it up and the user should be able to signup with just their firstname if they want i think.

**Pull request merged**